CONTRIBUTING.md

```diff
@@ -33,12 +33,12 @@ on the new feature or improvement.
 
 ### Maven tips
 
-A `./mvnw clean install` runs basically "everything" except release/deployment stuff. This is often
+A `./mvnw --threads 1C clean install` runs basically "everything" except release/deployment stuff. This is often
 not necessary. Some tips:
 
-* `./mvnw -Dquickly` Just compiles code, no tests, does not build code under `ui/` and `perftest/`.
-* `./mvnw -DskipTests` Compiles everything, runs no tests.
-* `./mvnw -DskipITs` Compiles everything, runs unit tests, but no integration tests.
+* `./mvnw --threads 1C package -Dquickly` Just compiles code, no tests, does not build code under `ui/` and `perftest/`.
```

Comment:
using `package` here to just build the artifacts, not install them into the local m2 repo

Reply:
I'm not sure about recommending `package` as the default goal. I usually do `install -Dquickly` and then run individual tests in IDE... but I'm not sure if that workflow will work with `package`. WDYT?

Reply:
It's a matter of expectation I guess... if not running `install`, I think IDE will use over artifacts for dependencies from the local repo :thinking: I'm not 100% sure, though.

Reply:
What I mean to say is that it might be preferable to leave the goal out of that example (as before) and let the use choose the goal to use with the "speed up" options.

Reply:
i added a goal because as is the mvnw calls were invalid... so maybe we should just be explaining the speedup flags here, and leave out the `./mvnw <goal>` part completely?

Reply:
+1 to that from my POV :)

Reply:
ok i've done that in the new commit